libbeat/docs/command-reference.asciidoc

```diff
@@ -20,11 +20,11 @@
 :modules-command-short-desc: Manages configured modules
 :run-command-short-desc: Runs {beatname_uc}. This command is used by default if you start {beatname_uc} without specifying a command
 
-ifeval::["{has_ml_jobs}"=="yes"]
+ifdef::has_ml_jobs[]
```

Comment:
Can you share some background on how this is different from the previous true / false?

Reply:
In `libbeat/docs/security/basic-auth.asciidoc` `has_ml_jobs` is used as `ifdef`.  It seems we typically use `ifdef` over `ifeval` with a value so I sync'd in that direction.